### ISSUE TITLE
Should not use "bold" for "bright"

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#1184]
(Original issue 1480 created by None on 2018-10-24T19:30:58.065463+00:00)

With Kitty terminal, "bold" does not mean "bright", which results in Python comments not being visible with pdbpp for me.

I've not really investigated yet (i.e. dug into pygments code), but it is very likely that color8 should be used instead of color1 in bold etc.

Ref: https://github.com/kovidgoyal/kitty/issues/197 / https://github.com/kovidgoyal/kitty/issues/781#issuecomment-409384077

**Comment by Anteru** [comment]
(Original issue 1480 last updated on 2019-05-18T21:20:07.303972+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by blueyed on 2019-05-10T13:46:47.704002+00:00)

Apparently this was filed by me - just came across this again.

The following patch fixes it:

```
diff -r 82bcd77a0459 pygments/console.py
--- a/pygments/console.py       Thu May 09 07:36:54 2019 +0200
+++ b/pygments/console.py       Fri May 10 15:16:51 2019 +0200
@@ -30,7 +30,7 @@
 x = 30
 for d, l in zip(dark_colors, light_colors):
     codes[d] = esc + "%im" % x
-    codes[l] = esc + "%i;01m" % x
+    codes[l] = esc + "%im" % (60 + x)
     x += 1

 del d, l, x
```

Reference: [https://en.wikipedia.org/wiki/ANSI\_escape\_code#3/4\_bit](https://en.wikipedia.org/wiki/ANSI_escape_code#3/4_bit)

**Comment by Anteru** [comment]
(Original comment by blueyed on 2019-05-10T13:47:33.515880+00:00)

Just for searchability: it causes e.g. the TerminalFormatter to not display comments.

**Comment by Anteru** [comment]
(Original comment by Anteru on 2019-05-18T12:08:58.359861+00:00)

Don't use bold for bright text in the console formatter (fixes #1480.)

→ <<cset eea5abc9161a>>

**Comment by Anteru** [comment]
(Original comment by blueyed on 2019-05-18T21:20:07.293598+00:00)

@Anteru thanks!
